src/components/SelectInput/index.ts

```diff
@@ -75,6 +77,22 @@ export interface SelectInputArgs extends ElementArgs, IBindableArgs, IPlaceholde
      * The type of each value. Can be one of 'string', 'number' or 'boolean'. Defaults to 'string'.
      */
     type?: 'string' | 'number' | 'boolean';
+    /**
+     * The order that the options should be checked in to find a valid fallback option that isn't included in the disabledOptions object.
+     */
+    fallbackOrder?: any;
```

Comment:
is there some better type info than `any'?

Reply:
Thanks, yep this should be an array of strings.